java/VisualVM/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                VisualVM
```

Comment:
This line could be omitted if you replace _visualvm_ with _VisualVM_ in the github.setup line

Reply:
Maybe, but I live in a case-exact world and would like to keep that principle up, if you don't mind. :-)